src/js/tabs/motors.js

```diff
@@ -742,6 +751,19 @@ motors.initialize = async function (callback) {
                 }
             });
 
+            if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_46)) {
+                const dshotEdtElement = $('input[id="dshotEdt"]');
+
+                dshotEdtElement.prop('checked', FC.MOTOR_CONFIG.use_dshot_edt).trigger("change");
+                dshotEdtElement.on("change", function () {
+                    const value = dshotEdtElement.is(':checked');
+                    const newValue = (value !== FC.MOTOR_CONFIG.use_dshot_edt) ? 'On' : 'Off';
+
+                    self.analyticsChanges['DshotEdt'] = newValue;
+                    FC.MOTOR_CONFIG.use_dshot_edt = value;
+                });
```

Comment:
Not sure about the code style here.

Better to use `this` or similar way to get the info about the actual order, sometimes things get weird with closures in js.

Also, not sure what triggering `change` event and then adding `on` handler does here 🤔

Something like the following is a bit more streamlined/usual in js land
```suggestion
const dshotEditCheckbox = $('#dshotEdt');

dshotEditCheckbox.prop('checked', FC.MOTOR_CONFIG.use_dshot_edt)
    .on("change", function () {
        const isChecked = $(this).is(':checked');
        const newValue = isChecked !== FC.MOTOR_CONFIG.use_dshot_edt ? 'On' : 'Off';

        self.analyticsChanges['DshotEdt'] = newValue;
        FC.MOTOR_CONFIG.use_dshot_edt = isChecked;
    })
    .trigger("change");
```